src/Microsoft.OData.Edm/EdmUtil.cs

```diff
@@ -492,6 +500,66 @@ internal static string FullyQualifiedName(IEdmVocabularyAnnotatable element)
             return null;
         }
 
+        private static string FullyQualifiedName(IEdmAnnotationsTarget annotationsTarget)

```

Comment:
>private [](http://example.com/codeflow?start=8&length=7)

make it as internal and do fully tests

Reply:
This method is being called from another internal method. 
I added tests to cover this